### PR TITLE
NCL-2302: enabled non-categorized tests when running with -Pcontainer…

### PR DESCRIPTION
**Pull request by michalszynkiewicz**
…-tests

### CODE REVIEW

File: build-executor/src/test/java/org/jboss/pnc/executor/BuildExecutionTest.java [hunk 1]
```diff
@@ -180,34 +179,13 @@ public void shouldNotContinueBuildOnMavenError() throws ExecutorException, Inter
         assertNoState(statusChangedEvents, BuildExecutionStatus.COLLECTING_RESULTS_FROM_REPOSITORY_MANAGER);
     }
 
-    @Test
-    public void shouldNotDestroyEnvironmentWhenPodKeptAliveSet() throws ExecutorException, InterruptedException, TimeoutException {
```
Comment (michalszynkiewicz) [inline build-executor/src/test/java/org/jboss/pnc/executor/BuildExecutionTest.java]: the test was no longer valid (env destroying is skipped at a deeper level)

File: process-managers/pom.xml [hunk 2]
```diff
@@ -67,6 +67,11 @@
       <artifactId>javax.inject</artifactId>
       <scope>provided</scope>
     </dependency>
+
+    <dependency>
+      <groupId>junit</groupId>
+      <artifactId>junit</artifactId>
+    </dependency>
```
Comment (tremes) [inline process-managers/pom.xml]: <scope>test</scope>?

File: rest/src/test/java/org/jboss/pnc/rest/endpoint/BuildRecordEndpointTest.java [hunk 3]
```diff
@@ -105,58 +98,6 @@ public void shouldGetLogsWithContent() {
...
-
-        // then
-        SshCredentials sshCredentials = record.getSshCredentials();
-        assertThat(sshCredentials).isNotNull();
-        assertThat(sshCredentials.getCommand()).isEqualTo(sshCommand);
-        assertThat(sshCredentials.getPassword()).isEqualTo(sshPassword);
-    }
-
-    @Test
-    public void shouldNotReturnSshCredentialsForOtherUsers() {
```
Comment (tremes) [inline rest/src/test/java/org/jboss/pnc/rest/endpoint/BuildRecordEndpointTest.java]: These two tests are also not valid?

Comment (michalszynkiewicz) [inline rest/src/test/java/org/jboss/pnc/rest/endpoint/BuildRecordEndpointTest.java]: I'm changing the functionality there ATM. I'm adding them in a different place

File: process-managers/pom.xml (hunk 2 above)
Comment (michalszynkiewicz) [inline process-managers/pom.xml]: Yes, I missed that. Thanks.I'll add `<scope>test</scope>` in the parent

**Pull request merged**